### DISCUSSION TITLE
Importing existing world did not work as expected

### DISCUSSION THREAD

**Discussion by zupd** [mbround18/valheim-docker#76] [Q&A] [answered]
Hi there!
I tried to import an existing world from my friend's computer.
He had the .db / .fwl - files for his world on his computer, and sent them to me.

I copied them to the `tmp/saves/world/` folder, and did `docker-compose down` -> `docker-compose up -d --build` to rebuild it. 

When I connected to the server. It looked like a completely new world, as I spawned in the starting area, and the map had not been explored at all.

What is the correct way to do this? I will double check that the files I got are correct, and try this myself with my own world now.

Also, would this setup be able to use multiple worlds on one dedicated server?
And is there a faster way of switching worlds on a server without rebuilding everything? That takes some time :P

**Comment by mbround18** [comment]
Does the world names match? I believe they have to. Also, i dont think valheim supports loading multiple worlds normally. You can spin up multiple valheim servers or just switch the world name thats provided.

**Reply by zupd** [reply]
The quickest way I have found to switch worlds is updating the yaml file and then run `docker-compose down` and `docker-compose up -d --no-deps --build valheim`

**Comment by zupd** [comment, marked as answer]
I tried with my own save files, and it worked fine. I believe that I have gotten the wrong files from my friend
